src/components/cart/Cart.jsx

```diff
@@ -65,33 +72,80 @@ function Cart() {
                       <header class="card-header">
                         <h3>{title}</h3>
                         <h4>
-                        &#8377; {price} &nbsp; <span class="text-strike"> &#8377;({original_price})</span> &nbsp; <span>({Math.trunc(calculateDiscount(price,original_price))}%)</span>
+                          &#8377; {price} &nbsp;{" "}
```

Comment:
Just as suggestion: Avoid using nbsp and {" "}, instead use elements for new line and css for spacing.